### PR TITLE
K7

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson4/backend28.js [hunk 1]
```diff
@@ -3,9 +3,11 @@ const express= require('express')
...
 app.listen(3618);
 app.use(express.static('public'));
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
 
-app.get('/send',(req,res)=>{
-  const firstdata = req.query.userNames;
-  const seconddata = req.query.userComments;
+app.post('/send',(req,res)=>{
+  const firstdata = req.body.name;
```
Comment (allopez7) [inline lesson4/backend28.js]: name the constants name and comment.

File: lesson4/backend28.js [hunk 2]
```diff
@@ -3,9 +3,11 @@ const express= require('express')
...
+  const seconddata = req.body.comment;
   fs.appendFile('/home/kinue/garageScript/lesson4/public/storing.txt', firstdata + ':' + seconddata + '\n')
```
Comment (evo777) [inline lesson4/backend28.js]: Use back ticks like this: `${firstData}: ${seconddata}` + '\n'

File: lesson4/backend28.js [hunk 3]
```diff
@@ -3,9 +3,11 @@ const express= require('express')
...
 app.use(express.static('public'));
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
 
-app.get('/send',(req,res)=>{
-  const firstdata = req.query.userNames;
-  const seconddata = req.query.userComments;
+app.post('/send',(req,res)=>{
+  const firstdata = req.body.name;
+  const seconddata = req.body.comment;
```
Comment (evo777) [inline lesson4/backend28.js]: I would change `seconddata` to `secondData` to make it camel case. If you do that, change all `seconddata` to camel case.

**Review by allopez7 — APPROVED** [review]
Good Job!

File: lesson4/backend28.js (hunk 3 above)
Comment (kinuesantos) [reply lesson4/backend28.js]: done, Thanks! :)

File: lesson4/backend28.js (hunk 2 above)
Comment (kinuesantos) [reply lesson4/backend28.js]: this works as well, right?   

`${firstData} : ${secondData}\n`

Comment (evo777) [reply lesson4/backend28.js]: Hmmm. I don't think so because the `\n` will be part of the string.

**Review by evo777 — APPROVED** [review]

**Pull request merged**